snuba/clickhouse/query.py

```diff
@@ -14,36 +16,33 @@ class ClickhouseQuery:
 
     def __init__(self,
         dataset: Dataset,
-        # TODO: pass query only once util.column_expr will not depend on the full request
-        # body anymore.
-        request: Request,
+        query: Query,
+        settings: RequestSettings,
```

Comment:
Starting to wonder if `RequestSettings` are named appropriately here…

Reply:
@manuzope you may want to look into how we are using them now.